src/SAML2/WebSSO/API.hs

```diff
@@ -121,18 +124,18 @@ instance FromMultipart Mem AuthnResponseBody where
                   lookupInput "SAMLResponse" resp
         parseAuthnResponseBody (cs base64)
 
+issuerToCreds :: forall m err. SPStoreIdP (Error err) m => Maybe Issuer -> m (NonEmpty SignCreds)
+issuerToCreds Nothing = throwError BadSamlResponseIssuerMissing
+issuerToCreds (Just issuer) = do
+    certs <- (^. idpMetadata . edCertAuthnResponse) <$> getIdPConfigByIssuer issuer
```

Comment:
Next time please use `view` instead of partially applied `^.` if you don't mind 😄